scripts/Jenkinsfile_tests_via_kcfg

```diff
@@ -0,0 +1,112 @@
+#!groovy
+
+node {
+  def branch = "main"
+  def test_dir = "${env.TEST_DIRECTORY_LIST}".split(',')
+
+  if ("${env.BRANCH}" != "null" && "${env.BRANCH}" != "") {
+    branch = "${env.BRANCH}"
+  }
+
+  def repo = scm.userRemoteConfigs
+  if ("${env.REPO}" != "null" && "${env.REPO}" != "") {
+    repo = [
+      [url: "${env.REPO}"]
+    ]
+  }
+  def timeout = "60m"
+  if ("${env.TIMEOUT}" != "null" && "${env.TIMEOUT}" != "") {
+    timeout = "${env.TIMEOUT}"
+  }
+  wrap([$class: 'AnsiColorBuildWrapper', 'colorMapName': 'XTerm', 'defaultFg': 2, 'defaultBg': 1]) {
+    withFolderProperties {
+      paramsMap = []
+      params.each {
+        if (it.value && it.value.trim() != "") {
+          paramsMap << "$it.key=$it.value"
+        }
+      }
+      withEnv(paramsMap) {
+        stage('Checkout') {
+          deleteDir()
+          checkout([
+            $class: 'GitSCM',
+            branches: [
+              [name: "*/${branch}"]
+            ],
+            extensions: scm.extensions + [
+              [$class: 'CleanCheckout']
+            ],
+            userRemoteConfigs: repo
+          ])
+        }
+        dir("./") {
+          stage('Run Sequential Jobs') {
+            echo "Running ${test_dir.size()} tests: ${test_dir}"
+
+            test_dir.each { test_name ->
+
+              def install_params = [
+                string(name: 'TEST_ARGS', value: "${TEST_ARGS}"),
+                string(name: 'REPO', value: "${REPO}"), 
+                string(name: 'BRANCH', value: "${BRANCH}"),                                         
+                string(name: 'PRODUCT_NAME', value: "${PRODUCT_NAME}"),                    
+                string(name: 'INSTALL_VERSION', value: "${INSTALL_VERSION}"),
+                string(name: 'INSTALL_MODE', value: "${INSTALL_MODE}"),
+                string(name: 'INSTALL_CHANNEL', value: "${INSTALL_CHANNEL}"),              
+                string(name: 'HOSTNAME_PREFIX', value: "${HOSTNAME_PREFIX}"), 
+                string(name: 'TEST_DIRECTORY', value: test_name),
+                string(name: 'AWS_SSH_KEY_NAME', value: "${AWS_SSH_KEY_NAME}"),
+                string(name: 'AWS_SSH_PEM_KEY', value: "${AWS_SSH_PEM_KEY}"),
+                string(name: 'KUBECONFIG', value: "${KUBECONFIG}"),                                                           
+              ]
+
+              switch(test_name) {
+                case "validatecluster":
+                  try {
+                    build job: "${env.PRODUCT_NAME}_validate_cluster", parameters: install_params
+                  } catch(err) {
+                    echo "Validate Cluster Test Error: " + err
+                    currentBuild.result = 'SUCCESS'
```

Comment:
Should we be setting this value as `SUCCESS`

Reply:
if i catch and set it temporarily, we wont be able to proceed with the test. like a failure will stop us from running the rest of the tests. am thinking ll set some other var and check the finally to reset it maybe.